src/styles/stars.scss

```diff
@@ -0,0 +1,23 @@
+.stars {
+  &--4 {
+    display: flex;
+    gap: 2px;
+    align-items: center;
+    padding-left: $padding-left;
```

Comment:
The variable `$padding-left` is used here, but it's not defined within this file. Ensure that it is defined in an imported file or within this file to avoid compilation errors.